okapi-core/src/main/java/org/folio/okapi/service/impl/DockerModuleHandle.java

```diff
@@ -402,13 +399,11 @@ private int getExposedPort(JsonObject b) {
       throw (new IllegalArgumentException(messages.getMessage("11301")));
     }
     int exposedPort = 0;
-    Iterator<Map.Entry<String, Object>> iterator = exposedPorts.iterator();
-    while (iterator.hasNext()) {
-      Map.Entry<String, Object> next = iterator.next();
+    for (Map.Entry<String, Object> next : exposedPorts) {
       String key = next.getKey();
       String port = key.split("/")[0];
       if (exposedPort == 0) {
-        exposedPort = Integer.valueOf(port);
+        exposedPort = Integer.parseInt(port);
       }
     }
     return exposedPort;
```

Comment:
for (Map.Entry<String, Object> next : exposedPorts) {
      String key = next.getKey();
      String port = key.split("/")[0];
      return Integer.parseInt(port);
    }
    return 0;

Reply:
ok.